sound/soc/sof/intel/pci-tgl.c

```diff
@@ -33,18 +33,18 @@ static const struct sof_dev_desc tgl_desc = {
 	.dspless_mode_supported	= true,		/* Only supported for HDaudio */
 	.default_fw_path = {
 		[SOF_IPC] = "intel/sof",
-		[SOF_INTEL_IPC4] = "intel/avs/tgl",
+		[SOF_INTEL_IPC4] = "intel/sof-ipc4/tgl",
```

Comment:
> I think you should clarify the backwards-compatibility story and which versions of sof-bin this will be aligned with. It wouldn't hurt to clarify why the change is not made for earlier cAVS versions

I think the commit  message says that the SOF firmware supports building for cAVS2.5 platforms, that is the reason why older platforms remain unchanged.
 
> Also if TGL uses this 'sof-ipc4-tplg' convention, I am not sure why we have the 'sof-ace-tplg'. It's a bit inconsistent, so my vote is to change MTL as well.

Right, but should that be included in this PR?